src/components/dashboard/DashboardPage.jsx

```diff
@@ -79,22 +108,7 @@ const DashboardPage = () => {
           {userFirstName ? `Welcome, ${userFirstName}!` : 'Welcome!'}
         </h2>
         <Tabs defaultActiveKey="courses">
-          <Tab eventKey="courses" title="Courses">
-            {CoursesTabComponent}
-          </Tab>
-          <Tab eventKey="programs" title="Programs">
-            <ProgramListingPage />
-          </Tab>
-          {features.FEATURE_ENABLE_PATHWAY_PROGRESS && (
-            <Tab eventKey="pathways" title="Pathways">
-              <PathwayProgressListingPage />
-            </Tab>
-          )}
-          {features.FEATURE_ENABLE_MY_CAREER && (
-            <Tab eventKey="my-career" title="My Career">
-              <MyCareerTab />
-            </Tab>
-          )}
+          {tabs.map(tab => (<Tab key={tab.props.eventKey} {...tab.props}>{tab.props.children}</Tab>))}
```

Comment:
You should be able to simply render `{tabs}` here rather than iterating over it. You're already defining each `Tab` component within `tabs` directly 😄

Reply:
😅